python/paddle/distributed/__init__.py

```diff
@@ -55,3 +56,10 @@
 
 # collective apis
 __all__ += collective.__all__
+'''
+__all__ = [
+    'spawn', 'init_parallel_env', 'get_rank', 'get_world_size',
+    'InMemoryDataset', 'QueueDataset', 'broadcast', 'all_reduce', 'reduce',
+    'all_gather', 'scatter', 'barrier', 'ReduceOp', 'FileInstantDataset',
```

Comment:
some of these, e.g.,  `all_reduce` should be include by  `__all__ += collective.__all__`